File: Learning_Tracker/Learnin_Traacker.API/Models/Note.cs
namespace Learnin_Traacker.API.Models
{
    public class Note
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public string Category { get; set; }
        public DateTime CreatedAt { get; set; }
        public string FirebaseUid { get; set; }



    }
}

File: Learning_Tracker/Learnin_Traacker.FRONT/learning-tracker/src/app/features/notes/note.ts
export interface Note {
  id: number;
  title: string;
  content: string;
  category: string;
  createdAt: string;
  firebaseUid: string;
}

export interface CreateNoteDto {
  title: string;
  content: string;
  category: string;
  firebaseUid: string;
}

export interface UpdateNoteDto {
  title: string;
  content: string;
  category: string;
}
